app/src/main/java/com/peacecorps/pcsa/safety_tools/SafetyToolsFragment.java

```diff
@@ -0,0 +1,100 @@
+        radarButton = (Button)rootView.findViewById(R.id.radarButton);
+        unwantedAttentionButton = (Button)rootView.findViewById(R.id.unwantedAttentionButton);
+        tacticsButton = (Button)rootView.findViewById(R.id.tacticsButton);
+        bystanderButton = (Button)rootView.findViewById(R.id.bystanderButton);
+        safetyPlanButton = (Button)rootView.findViewById(R.id.safetyPlanButton);
+        safetyPlanBasicsButton = (Button)rootView.findViewById(R.id.safetyPlanBasicsButton);
+
+        radarButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                //Swapping RadarFragment into the container
+                RadarFragment radarFragment = new RadarFragment();
+                MainActivity.swapFragmentIn(getActivity(),radarFragment,RadarFragment.TAG,true);
+            }
+        });
+
+        unwantedAttentionButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                //Swapping UnwantedAttentionStrategies into the container
+                CopingFragment radarFragment = new CopingFragment();
+                MainActivity.swapFragmentIn(getActivity(),radarFragment,CopingFragment.TAG,true);
+            }
+        });
+
+        tacticsButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                //Swapping TacticsFragment into the container
+                TacticsFragment tacticsFragment = new TacticsFragment();
+                MainActivity.swapFragmentIn(getActivity(),tacticsFragment,TacticsFragment.TAG,true);
+            }
+        });
+
+        bystanderButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                //Swapping Bystander Fragment into the container
+                BystanderInterventionFragment bystanderInterventionFragment = new BystanderInterventionFragment();
+                MainActivity.swapFragmentIn(getActivity(),bystanderInterventionFragment,BystanderInterventionFragment.TAG,true);
+            }
+        });
+
+        safetyPlanBasicsButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                //Swapping Safety Plan Basics Fragment into the container
+                SafetyPlanBasicsFragment safetyPlanBasicsFragment = new SafetyPlanBasicsFragment();
+                MainActivity.swapFragmentIn(getActivity(),safetyPlanBasicsFragment,SafetyPlanBasicsFragment.TAG,true);
+            }
+        });
+
+        safetyPlanButton.setOnClickListener(new View.OnClickListener() {
+            @TargetApi(Build.VERSION_CODES.JELLY_BEAN)
+            @Override
+            public void onClick(View v) {
+                //Swapping Safety Plan Fragment into the container
```

Comment:
Missed this, sorry about it. 
It's launching another activity, the comment says otherwise